### ISSUE TITLE
Seeking new maintainer(s) for mastodon-twitter-sync

### ISSUE THREAD

**Issue by klausi** [klausi/mastodon-twitter-sync#80]
It finally happened. Twitter suspended my Mastodon Twitter Sync app for terms of service violations.

Quickly reviving it with a new app did not work, so I think this could be the end of this sync project for me.

As I'm using Twitter less and less I'm not keen on continuing any effort on it ... anyone up for adopting mastodon-twitter-sync?

I will probably archive this repository soon and am happy to point users to a new home if somebody would like to take over.

**Comment by orhun** [comment]
Hey, sad to hear that they suspended the app. I'm still using `mastodon-twitter-sync` on my homeserver (it works like a charm) and I would be happy to take over maintainership!

Thanks for creating this great project! 💖

**Comment by klausi** [comment]
Nice, thanks for stepping up! @nfrankel also showed interest, maybe check with him what would be best for a new home.

To preserve history and past ownership I would like to freeze and archive this repo under my username. Feel free to fork or start from scratch, then post the link to the new home here and I'll gladly point to it from the README.

**Comment by nfrankel** [comment]
I don't have a problem for the new home if @orhun is interested in hosting it.

Sorunumuz yok!

**Comment by orhun** [comment]
Good news, I created the organization/repo: https://github.com/mastodon-twitter-sync/mastodon-twitter-sync 🎉

Also sent you guys an invite!

Soon I will transferring the issues/PRs and start working on a maintenance release. Will keep you updated! 🐻

**Comment by klausi** [comment]
Thanks a lot!

I was thinking if I should switch this project from Twitter API to Twitter scraping with https://github.com/megahotel/twitter-scraper or similar, but did not find the motivation and time yet.

Anyway, having the current mastodon-twitter-sync under a broader umbrella is a good step!

I will archive this project in a couple of days and point to the new home.

**Comment by orhun** [comment]
Bad news.

![Screenshot 2023-06-19 at 13-06-40 Twitter Developers](https://github.com/klausi/mastodon-twitter-sync/assets/24392180/ecdfa161-1323-42a7-8713-86d3aff6be58)

> I was thinking if I should switch this project from Twitter API to Twitter scraping with [megahotel/twitter-scraper](https://github.com/megahotel/twitter-scraper) or similar, but did not find the motivation and time yet.

I think that's the first thing that I am going to do now 😄

**Comment by winguse** [comment]
yeah, I find the same thing for me. so I guess the thing finally comes to an end. Thank you so much!

**Comment by orhun** [comment]
I stepped up for maintainership in the first place with the hopes that I can use an alternative solution to fetch tweets etc. but with the latest changes in X, I semi-lost my motivation. If anyone wants to take ownership of the organization that I created just let me know.

P.S. Came across to a solution called [Buffer](https://buffer.com/) which can be potentially used for cross-posting.

**Comment by nfrankel** [comment]
I'm crossposting manually as of now, so thanks but no thanks.
